google/api_core/exceptions.py

```diff
@@ -476,22 +476,31 @@ def from_http_status(status_code, message, **kwargs):
     return error
 
 
-def from_http_response(response):
-    """Create a :class:`GoogleAPICallError` from a :class:`requests.Response`.
+def _format_error_message(error, method, url):
+    method = method.upper()
+    message = "{method} {url}: {error}".format(
+        method=method,
+        url=url,
+        error=error,
+    )
+    return message
+
+
+def format_http_response_error(response, method, url, payload=None):
```

Comment:
nit: add type hints 
nit: add code comments to clarify the reason that we're moving away from `from_http_response`

Reply:
addressed with a comment.